adoption.rst

```diff
@@ -79,6 +79,7 @@ bodies of documentation. In some cases the adoption remains partial or is still
 * `StrongLoop/LoopBack <https://loopback.io/doc/en/lb4>`_ by IBM
 * `TerminusDB <https://terminusdb.com/docs/terminusdb/#/>`_
 * Tesla Motors (internal)
+* `Wikimedia Foundation <https://www.wikimedia.org/>`_, technical documentation for Wikimedia projects and open-source software
```

Comment:
Did you mean to link to https://www.mediawiki.org? I don't see any technical documentation at https://www.wikimedia.org, let alone docs conforming to the Diátaxis framework.

Reply:
Because the technical documentation for Wikimedia projects is scattered across many wikis and other platforms, it seemed most succinct to link to that page because it links to all the many projects.  I guess I also thought the goal was to link to the organization's website, not a specific set of documentation.  That said, maybe a link to [developer.wikimedia.org](http://developer.wikimedia.org) would be better?

Reply:
That's great to hear! So I probably would suggest linking to the toolkit page in this PR (or better yet, the ["Templates for common document types" section](https://www.mediawiki.org/wiki/Documentation/Toolkit#Templates_for_common_document_types) directly), unless there is a particular instance of Wikimedia docs already structured that way that you think would be worth highlighting. Otherwise, just linking to wikimedia.org seems like it might confuse people who actually follow the link expecting to see usage of the framework there.